rag_experiment_accelerator/doc_loader/pdfLoader.py

```diff
@@ -49,7 +49,7 @@ def _load_pdf_with_pypdf(file_path: str) -> List[Document]:
 
 def _load_pdf_with_unstructured(file_path: str) -> List[Document]:
     logger.debug("Loading PDF file with unstructured: %s", file_path)
-    elements = partition_pdf(filename=file_path, strategy="ocr_only")
+    elements = partition_pdf(filename=file_path, strategy="hi_res")
     element_meta = elements[0].to_dict()["metadata"] if elements else {}
```

Comment:
Severity: Medium - The strategy for partitioning the PDF content has been updated from 'ocr_only' to 'hi_res'. Ensure that this change aligns with your application's requirements and performance goals. If applicable, consider adding a comment to clarify why 'hi_res' is the preferred option.


[Hack - peer-review-pal] We are sorry for the intrusion. Feel free to ignore/delete the comments with the tag - they are automatically generated by [our Hack project](https://github.com/microsoft/peer-review-pal).
If you find them useful, want us to delete them or have any feedback or know how to make it better and want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding #prp-ignore to your PR description.